podcasts/EpisodeDetailViewController.swift

```diff
@@ -500,3 +567,44 @@ enum EpisodeDetailViewSource: String, AnalyticsDescribable {
 
     var analyticsDescription: String { rawValue }
 }
+
+// MARK: - UIScrollView Extension
+
+private extension UIScrollView {
+    /// Adds the view to the scroll view and sets its constraints
+    /// If `after` is specify the `view` will be aligned horizontally after the `after` view
+    func addPageView(_ view: UIView, after: UIView? = nil) {
```

Comment:
I wonder whether is a better option to inherit from `UIScrollView` and create a `UIPagedScrollView`?

This wouldn't change much of the code but it will be very declarative that it's a `UIScrollView` with Paging capabilities.

Reply:
Good idea, I moved the relevant code into a new `PagedUIScrollView` class.